Framework/DataHandling/test/ISISJournalTest.h

```diff
@@ -0,0 +1,216 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#pragma once
+
+#include <cxxtest/TestSuite.h>
+
+#include "MantidDataHandling/ISISJournal.h"
+
+using namespace Mantid::DataHandling::ISISJournal;
+
+class ISISJournalTest : public CxxTest::TestSuite {
+public:
+  // This pair of boilerplate methods prevent the suite being created statically
+  // This means the constructor isn't called when running other tests
+  static ISISJournalTest *createSuite() { return new ISISJournalTest(); }
+  static void destroySuite(ISISJournalTest *suite) { delete suite; }
+
+  void test_getRunData_with_empty_file_throws() {
+    TS_ASSERT_THROWS(
+        getRunDataFromFile(emptyFile, defaultTags(), defaultFilters()),
+        std::runtime_error);
```

Comment:
All of these are catching a copy since the type is copied in by CXXtest, which emits warnings with GCC-8  onwards
Can you mark all of them `std::runtime_error &`